src/lib/annotations/doc/CreateHighlightDialog.js

```diff
@@ -173,6 +173,7 @@ class CreateHighlightDialog extends EventEmitter {
         }
 
         hideElement(this.containerEl);
+        document.getSelection().removeAllRanges();
```

Comment:
Can you please move this into the highlight annotator? I want to keep this a dumb piece of UI, nothing more than that.

Reply:
Moved, had to move it to two places :(. One where we enter another mode, and one for the mouseupHandler for clicking the document.